lib/pbench/server/database/alembic/env.py

```diff
@@ -0,0 +1,107 @@
+"""
+This file is auto generated when we run `alembic init alembic` but modified according to our needs.
+This Python script runs whenever the alembic migration tool is invoked.
+It contains instructions to configure and generate a SQLAlchemy engine,
+procure a connection from that engine along with a transaction, and then
+invoke the migration engine, using the connection as a source of database connectivity.
+"""
+import sys
+from logging.config import fileConfig
+
+
+from alembic import context
+
+from pbench.common.exceptions import BadConfig, ConfigFileNotSpecified
+from pbench.server.database.database import Database
+from pbench.common.logger import get_pbench_logger
+from pbench.server.api import get_server_config
+
+# this is the Alembic Config object, which provides
+# access to the values within the .ini file in use.
+config = context.config
+
+# Interpret the config file for Python logging.
+# This line sets up loggers basically.
+fileConfig(config.config_file_name)
```

Comment:
Do we have a separate configuration file for Alembic?

Reply:
For now we are just using the default alembic logging that comes with `alembic init`. This results in logs being produced via `sys.stderr` when running alembic from console and the configuration is written in `alembic.ini` file. However, now that you mentioned it I added the `fileHandler` settings in the `alembic.ini` file that will write the migration logs to a specified file in `/var/log` directory along with console output.

Reply:
The pbench server does not log to /var/log for anything else.  This would be a first.  Instead, it logs through /dev/log which goes to systemd/journald.  So if we could do that instead to match, that would be great.

Reply:
fixed